plotly/express/_chart_types.py

```diff
@@ -5727,7 +5727,7 @@ def choropleth_mapbox(
 ) -> go.Figure:
     """
     *choropleth_mapbox* is deprecated! Use *choropleth_map* instead.
-    Learn more at: https://plotly.com/python/mapbox-to-maplibre/
+    Learn more at: https://plotly.com/python/mapbox-to-maplibre/.
```

Comment:
Might the `.` break the URL when you view it in the terminal?

Reply:
They don't because these don't render as links in markdown (I haven't resolved it because there is no `mapbox-to-maplibre` page, but we can exclude the `'.'` in the actual link later on.